services/datamanager/builtin/builtin.go

```diff
@@ -505,7 +505,7 @@ func (svc *builtIn) sync() {
 	}
 }
 
-// nolint
+//nolint
```

Comment:
Wait, this is super ironic.  The linter linted a comment telling it not to lint

Reply:
Yeah, it does that. It occasionally breaks things when it does too.